azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/AbstractAzureMojo.java

```diff
@@ -366,24 +374,11 @@ protected boolean isAuthConfigurationExist() {
         map.put(PLUGIN_VERSION_KEY, getPluginVersion());
         map.put(SUBSCRIPTION_ID_KEY, getSubscriptionId());
         map.put(SESSION_ID_KEY, getSessionId());
-        map.put(AUTH_TYPE, getAuthType());
         return map;
     }
 
-    // TODO:
-    // Add AuthType ENUM and move to AzureAuthHelper.
     public String getAuthType() {
-        final AuthenticationSetting authSetting = getAuthenticationSetting();
-        if (authSetting == null) {
-            return "AzureCLI";
-        }
-        if (StringUtils.isNotEmpty(authSetting.getServerId())) {
-            return "ServerId";
-        }
-        if (authSetting.getFile() != null) {
-            return "AuthFile";
-        }
-        return "Unknown";
+        return azureTokenWrapper == null ? AUTH_TYPE_UNKNOWN : azureTokenWrapper.getAuthType().name();
```

Comment:
AUTH_TYPE_UNKNOWN is confusing here

Reply:
Fall back to old codes here to adapt users who use old authentication methods